person.rb

```diff
@@ -0,0 +1,24 @@
+# implementation person class
+class Person
+  def initialize(age, _parent_permission: true, name: 'Unknown')
+    @id = id
+    @name = name
+    @age = age
+  end
+
+  # Getter for id
+  attr_reader :id
+  # Getters/setters for name and age
+  attr_accessor :name
+  attr_accessor :age
+
+  def of_age?
+    @age >= 18
+  end
```

Comment:
- [ ] `of_age?` is a private  that returns true if @age is greater or equal to 18 and false otherwise. Please make the method private. Something like so:
```ruby
  private 
  def is_of_age?
    @age >= 18
  end
```

- [ ] `can_use_services?` is public method, please kindly fix that.